Update surveymonkey.md (#39579)
airbytehq/airbyte: docs/integrations/sources/surveymonkey.md

> # Set up the SurveyMonkey Source Connector

Reviewer: ```suggestion
# SurveyMonkey

This page guides you through setting up the SurveyMonkey source connector.
```
```suggestion
# Set up the SurveyMonkey Source Connector
```

Reviewer: This is where the navigation bar gets the name of the source to include in the source catalog. As is, when you look for this source in the sources list, it's listed as "Set up the SurveyMonkey Source Connector".

Reviewer: ![Screenshot 2024-06-24 at 11 56 32 AM](https://github.com/airbytehq/airbyte/assets/23410478/204e5a69-cd83-413c-9666-614a38794e04)

<!-- env:oss -->

## Prerequisites

**SurveyMonkey:**
- Registered Private application [here](https://developer.surveymonkey.com/apps/)
- SurveyMonkey authorization credentials
- SurveyMonkey authorization method (oauth2.0)

> 

Reviewer: :::info
To learn more about setup and requirements associated with Private apps on SurveyMonkey, visit their documentation [here](https://api.surveymonkey.com/v3/docs#private-apps)
:::

Reviewer: There are potential limits and account type requirements that we might not want to enumerate, but that we should guide users to be able to find easily.

> **For Airbyte Open Source:**

Reviewer: ```suggestion
```

> - Access Token

Reviewer: ```suggestion
- Access Token (required in Open Source only)
```

> - Start date for the Source Connector (in UTC)

Reviewer: I looked at the setup fields in both Open Source and Cloud. Start date is required on any of the platforms. The only distinction for Open Source is that we need an Access Token. I'll suggest one was we can make this clearer.

<!-- /env:oss -->

## Step 1: Set up SurveyMonkey

1. Log in to SurveyMonkey.

> 2. Then go to Settings and copy the Private app access token.

Reviewer: ```suggestion
2. From [your list of Apps](https://developer.surveymonkey.com/apps/) on SurveyMonkey, navigate to Settings for your Private app, copy the Access Token.
```

Reviewer: Important because without this clarification, users may log into SurveyMonkey and discover there is nothing labeled "Settings"...which they will only find in association with the developer.surveymonkey.com/apps page.

## Step 2: Set up the source connector in Airbyte
SurveyMonkey Privaye Apps are supported in both Airbyte Cloud and Airbyte Open Source.
<!-- env:cloud -->

**Set up a SurveyMonkey Source Connector in Airbyte Cloud:**

1. [Log in to Airbyte Cloud](https://cloud.airbyte.com/workspaces).
2. Click **Sources**. Click ** new source**.
3. In **Source Setup**, select **SurveyMonkey** from **Source type**.
4. Enter a name for this **Source Connector**.
5. Click **Authenticate your account**.

> 6. Log in and authorize to **SurveyMonkey**.

Reviewer: ```suggestion
6. Log in to authenticate **SurveyMonkey**.
```

7. Choose a **Start date**.
8. Click **Set up source**.
<!-- /env:cloud -->

<!-- env:oss -->

**Set up a SurveyMonkey Source Connecor in Airbyte Open Source:**

1. Go to local Airbyte page.
2. Click **Sources**. Click **New Source**.
3. In **Source setup**, select **SurveyMonkey** from **Source type**.
4. Enter a **Name** for the new Source Connector.

> 5. Add **Access Token**

Reviewer: ```suggestion
5. Add your **Access Token**
```
